frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelCorrelationTable.scss

```diff
@@ -20,6 +20,7 @@
         border-top-left-radius: $radius;
         border-top-right-radius: $radius;
         padding-bottom: 5px;
+        overflow: auto;
```

Comment:
@Twixes do you remember why this was necessary? Seems to be the cause of the hidden popup

Reply:
Hmm, I think something was overlaid in a weird way without this, but honestly not sure at the moment. 🤔